platform/core/src/types/index.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  StudyMetadata,
+  SeriesMetadata,
+  InstanceMetadata,
+} from './StudyMetadata';
+import Params from './ParamsType';
+import Consumer from './Consumer';
+
+// These are just Params right now, but eventually the idea is to
+// create a full interface object here - at that point they should
+// move to their own files.
+export type ICommandsManager = Params;
+
+export type IGuiCustomization = Params;
```

Comment:
`ICommandsManager = Params` again is so weird, lets just put it as `unkown` or `any` or at worst the `Record<string, unknown>` until we type it

Reply:
Also the IGuiCustomization stuff is not merged yet so let's remove it too